CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
    - Document `doc/ellswift.md` which explains the mathematical background of the scheme.
    - The [paper](https://eprint.iacr.org/2022/759) on which the scheme is based.
 
+#### Changed
+ - The user must define the `SECP256K1_STATIC` macro when consuming libsecp256k1 as a static library for their Windows builds.
```

Comment:
nit:
```suggestion
 - The user must now define the `SECP256K1_STATIC` macro before including `secp256k1.h` when consuming libsecp256k1 as a static library for their Windows builds.
```

Or stylistic variant that is easier to parse:
```suggestion
 - When consuming libsecp256k1 as a static library on Windows, the user must now define the `SECP256K1_STATIC` macro before including `secp256k1.h`.
```

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/secp256k1/pull/1367#issuecomment-1614333593).